docs/docs-site/content/developer/api/rest/_index.md

```diff
@@ -441,7 +441,14 @@ Upload a local file to a remote destination folder:
 
 ### File import
 
-First guessing the format of the file `s3a://demo-hue/web_log_data/index_data.csv`:
+We have 2 options here.
+
+#### Remote file (S3/HDFS file)
```

Comment:
Simpler titles? (no S3 etc...)

No Ayush in the example path?

Reply:
Use two bullet points?

Explain that two parameters differ?

Reply:
![Screenshot 2021-07-22 at 6 59 42 PM](https://user-images.githubusercontent.com/36241930/126647199-026f99fe-986f-4666-8646-ddc1407bdfa4.png)


So now it is good or other improvements needed. @romainr